test/lib/ufe/testDuplicateCmd.py

```diff
@@ -47,7 +47,11 @@ class DuplicateCmdTestCase(unittest.TestCase):
     def setUpClass(cls):
         if not cls.pluginsLoaded:
             cls.pluginsLoaded = mayaUtils.isMayaUsdPluginLoaded()
-    
+
+    @classmethod    
```

Comment:
Unused?

Reply:
That's a workflow error on my part.  Trying to juggle too many tasks this morning.  Let me clean that up before you review anymore.

Reply:
:)  No trouble, cheers.